python-sdk/nuscenes/eval/detection/main.py

```diff
@@ -129,10 +136,29 @@ def run(self) -> DetectionMetrics:
 
         return metrics
 
+    def render(self, md_list: MetricDataList):
```

Comment:
Maybe `NuScenesEval.run()` should return both `DetectionMetrics` and `MetricDataList` so that the output of run() can be passed to render().